index.html

```diff
@@ -0,0 +1,231 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css"
+      integrity="sha512-Fo3rlrZj/k7ujTnHg4CGR2D7kSs0v4LLanw2qksYuRlEzO+tcaEPQogQ0KaoGN26/zrn20ImR1DfuLWnOo7aBA=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="./styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <main class="first-card">
+      <section class="first-card-section">
+        <div class="all-components">
+          <div class="first-image">
+            <img src="./asset/Snapshoot.png" alt="" />
+          </div>
+          <div class="tonic-content">
+            <h1>TONIC</h1>
```

Comment:
- [ ] Kindly make sure you have only one `h1` on your page as it helps the SEO(search engine optimization) find your page faster having multiple will cause a lot of confusion👍